app/src/ui/app.tsx

```diff
@@ -583,6 +585,23 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClickDarwin() {
+    const actionOnDoubleClick: AppleActionOnDoubleClickPref = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string')
```

Comment:
I'm generally always in favor of the compiler yelling at me, and with the dangerous nature of typing something as any it sure would be nice to be explicit about it. I wonder if there was a good reason why we didn't turn that particular flag to begin with. Paging @joshaber